src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -186,12 +164,59 @@ def create_chart_from_prometheus_query(
             values.append((timestamp, value))
         min_time = min(min_time, min(series.timestamps))
         max_time = max(max_time, max(series.timestamps))
-        chart.add(label, values)
+
+        plot_list.append((label, values))
+        graph_plot_color_list.append("#9747FF")
 
     assert lines is not None
     for line in lines:
         value = [(min_time, line.value), (max_time, line.value)]
-        chart.add(line.label, value)
+
+        if line.label == "Memory Limit" \
+                or line.label == "CPU Limit":
+            plot_list.append((line.label, value))
+            graph_plot_color_list.append("#FF5959")
+
+        elif line.label == "Memory Request" \
+                or line.label == "CPU Request":
+            plot_list.append((line.label, value))
+            graph_plot_color_list.append("#0DC291")
+
+        else:
+            plot_list.append((line.label, value))
+            graph_plot_color_list.append("#2a0065")
+
+    graph_plot_color_list.extend(["#1e0047", "#2a0065"])
+    chart = pygal.XY(
+        show_dots=True,
+        style=charts_style(graph_colors=tuple(graph_plot_color_list)),
+        truncate_legend=15,
+        include_x_axis=include_x_axis,
+        width=1280,
+        height=720,
+    )
+
+    chart.x_label_rotation = 35
+    chart.truncate_label = -1
+    chart.x_value_formatter = lambda timestamp: datetime.fromtimestamp(timestamp).strftime("%b %-d %H:%M")
+
+    value_formatters = {
+        ChartValuesFormat.Plain: lambda val: str(val),
+        ChartValuesFormat.Bytes: lambda val: humanize.naturalsize(val, binary=True),
+        ChartValuesFormat.Percentage: lambda val: f"{(100 * val):.1f}%",
+        ChartValuesFormat.CPUUsage: lambda val: f"{(1000 * val):.1f}m",
+    }
+    chart_values_format = values_format if values_format else ChartValuesFormat.Plain
+    chart.value_formatter = value_formatters[chart_values_format]
+
+    if chart_title:
+        chart.title = chart_title
+    else:
+        chart.title = promql_query
+
+    for plot in plot_list:
```

Comment:
what is this for?

Reply:
as discussed we need this plot_list for adding a line to the chart